modules/pam_env/pam_env.8.xml

```diff
@@ -163,6 +163,11 @@
             behavior of subsequent modules in the stack without the consent
             of the system administrator.
           </para>
+          <para>
+            Due to problematic security this functionality is deprecated
+            since the 1.5.0 version and will be removed completely at some
+            point in future.
```

Comment:
I'd suggest using "in the future" wording as in the NEWS entry.
Besides that, LGTM.

Reply:
Done